sources/zh.picacomic/src/settings.rs

```diff
@@ -0,0 +1,40 @@
+use aidoku::{
+	Result,
+	alloc::{String, Vec, string::ToString},
+	error,
+	imports::defaults::{DefaultValue, defaults_get, defaults_set},
+};
+
+pub fn get_app_channel() -> String {
+	defaults_get::<String>("appChannel").unwrap_or("3".into())
+}
+
+pub fn get_image_quality() -> String {
+	defaults_get::<String>("imageQuality").unwrap_or("original".into())
+}
+
+pub fn get_blocklist() -> Vec<String> {
+	defaults_get::<Vec<String>>("blockGenres").unwrap_or_default()
+}
+
+pub fn get_list_viewer() -> bool {
+	defaults_get("isListView").unwrap_or(false)
+}
+
+pub fn get_username() -> Result<String> {
+	defaults_get::<String>("username").ok_or_else(|| error!("Please login first"))
```

Comment:
technically it should be "log in" as a verb, but does it make more sense to have error messages in chinese since it's a chinese source?
```suggestion
	defaults_get::<String>("username").ok_or_else(|| error!("Please log in first"))
```

Reply:
Fix, I have no intention of maintaining the Chinese error message of the Chinese source. So keep it as it is.